GeoFire.podspec

```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
```

Comment:
It doesn't require 2.2, but bugfixes are only released for the latest minor version.